bokeh/sphinxext/bokeh_plot.py

```diff
@@ -172,7 +175,10 @@ def run(self):
 
     def _get_source(self):
         if self.arguments:
-            source = open(self.arguments[0], "r").read()
+            path = self.arguments[0]
+            if path.startswith("docs"):
+                path = join("source", path)
+            source = open(path, "r").read()
```

Comment:
This is just some junk to make things work with the path renames below. I wanted to do all renames in one commit by themselves to make reviewing the later commits easier.